llama-index-core/llama_index/core/indices/property_graph/sub_retrievers/text_to_cypher.py

```diff
@@ -41,6 +66,8 @@ def __init__(
         response_template: Optional[str] = None,
         cypher_validator: Optional[Callable] = None,
         allowed_output_fields: Optional[List[str]] = None,
+        include_raw_response_as_metadata: Optional[bool] = False,
+        summarize_response: Optional[bool] = False,
```

Comment:
we should probably make the prompt customizable here

Reply:
Good call, I have added this as an optional prompt